### PR TITLE
Fixes #9019 - fixed N+1 query on dashboard widget

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: app/views/dashboard/_discovery_widget.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h4 class="ca"><%= _('Discovered Host Pool') %> - <%= @current_permission %></h4>
-<% all_hosts = Host::Discovered.search_for '' %>
+<% all_hosts = Host::Discovered.includes(:model).includes(:discovery_attribute_set).search_for '' %>
```
Comment (orrabin) [inline app/views/dashboard/_discovery_widget.html.erb]: I know this was in the original code but why do we do `.search_for ''` and not `.scoped`?
or just 

``` ruby
 <% hosts =Host::Discovered.includes(:model).includes(:discovery_attribute_set).limit(5) %>
```

Comment (orrabin) [inline app/views/dashboard/_discovery_widget.html.erb]: I didn't see the `all_hosts.size` at the end, but we can still change the search_for maybe?

**Comment by lzap** [comment]
Ok it works fine.

**Comment by orrabin** [comment]
merged as c46a16076a3487a9df331b4f7a62ab736970b36a thanks @lzap

**Pull request closed without merging**